src/components/ResourceList/index.ts

```diff
@@ -1,8 +1,9 @@
 import ResourceList from './ResourceList';
 
+export {Consumer} from './components';
 export * from './components/FilterControl/types';
 export {Props as FilterControlProps} from './components/FilterControl';
 export {Props} from './ResourceList';
-export {SelectedItems} from './types';
+export {SelectedItems, ResourceListContext} from './types';
```

Comment:
Exporting this to use the context in filters, specifically to get the `resourceName`